src/routes/tools/extreme-precipitation/_ExploreData.svelte

```diff
@@ -103,7 +104,7 @@
     } else {
       const [lng, lat] = $location.center;
       const modelsStr = $modelsStore.join(",");
-      bookmark = `climvar=${$climvarStore}&scenario=${$scenarioStore}&models=${modelsStr}&lng=${lng}&lat=${lat}&boundary=${$boundary.id}`;
+      bookmark = `climvar=${$climvarStore}&scenario=${$scenarioStore}&duration=${$durationStore}&threshId=${$thresholdTypeStore}&models=${modelsStr}&lng=${lng}&lat=${lat}&boundary=${$boundary.id}`;
```

Comment:
Currently when users copy paste the share link, the tool still loads the default parameters and not parameters in the share link. I will add a fix for this issue a separate PR since this behavior affects all tools.